keanu-python/keanu/vertex.py

```diff
@@ -109,8 +109,23 @@ def __init__(self, ctor, *args):
         super(Vertex, self).__init__(ctor, *(Vertex.__parse_args(*args)))
 
     def observe(self, v):
```

Comment:
nit: can you add some whitespace between `VertexOps` and the `Vertex` class?

Reply:
ok